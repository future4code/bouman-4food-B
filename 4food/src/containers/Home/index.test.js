import React from "react";
import { Home } from "./index";
import renderer from "react-test-renderer";

describe("Test Home", () =>{
    test("Snapshot Home", () => {
        const testHome = renderer.create(<Home/>)
        .toJSON();
        expect(testHome).toMatchSnapshot();
    });

});