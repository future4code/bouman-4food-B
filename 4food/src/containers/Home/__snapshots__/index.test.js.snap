// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Home Snapshot Home 1`] = `
<div
  className="sc-kGXeez dOkNpw"
>
  <div
    className="sc-kpOJdX gOEmaK"
  >
    <img
      className="sc-dxgOiQ hLmPai"
      src="logo-branco.svg"
    />
  </div>
</div>
`;
